rj_gameplay/rj_gameplay/role/runner_role.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+
+import stp
+from rj_gameplay.skill import move
+from rj_msgs.msg import RobotIntent
+import numpy as np
+
+
+class RunnerRole(stp.role.Role):
+    def __init__(self, robot: stp.rc.Robot) -> None:
+        super().__init__(robot)
+        self.move_skill = None
+        self.target_point = None
+        # self.robot_id = robot.id
+
+    def tick(self, world_state: stp.rc.WorldState) -> RobotIntent:
+        # current_pos = world_state.our_robots[self.robot_id].pose
+        if (self.move_skill is None or self.move_skill.is_done(world_state)):
+            self.target_point = self.next_point(world_state)
+        
+            # assign move skill
+            self.move_skill = move.Move(
+                robot=self.robot,
+                target_point=self.target_point,
+                face_point=self.target_point
+            )
+
+        intent = self.move_skill.tick(world_state)
+
+        return intent
+        
+        # world_state.field.our_left_corner
+    def is_done(self, world_state: stp.rc.WorldState) -> bool:
+        return False
+    
+    def next_point(self, world_state: stp.rc.WorldState) -> np.ndarray:
```

Comment:
I would add comments on what you are making so it's easier for the person who looks at your code to follow your process. Future tip!